### DISCUSSION TITLE
What is the best way to handle armv7hf and aarch64 for ACAP applications

### DISCUSSION THREAD

**Discussion by polodroid74** [AxisCommunications/discussions#253] [ACAP]
Hi,
As an ACAP developer, I have to develop applications that work on ARTPEC7 and 8 SOCs.
For now, the best way I've found to achieve this, is to have two docker images : One for armv7hf cross-compilation and one for armv8 cross-compilation.
This is working quite well but it is time consuming to handle. Indeed, when I create a new version of my application, I need to cross-compile my code and all the external libs twice using two docker images.

Is there a way to better handle this ? For example, can the aarch64 docker image handle the armv7hf architecture with some tweaks ?

For now i'm using ACAP 3.5 docker images.

Best Regards,
Paul

**Comment by vivekatoffice** [comment]
AXIS Camera Application Platform (ACAP) documentation: https://developer.axis.com/acap/
